frontend/src/app/redirect.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Inject, OpaqueToken } from '@angular/core';
+import { Router } from '@angular/router';
+
+import { environment } from '../environments/environment';
+
+export const WindowToken = new OpaqueToken('Window');
+
+export function _window(): Window {
+  return window;
+}
+
+@Component({
+  template: ''
+})
+export class RedirectComponent {
+  constructor(
+    private router: Router,
+    @Inject(WindowToken) private window: any
+  ) {
+    const angularUrlSuffix = this.router.routerState.snapshot.url;
+    const djangoUrlSuffix = angularUrlSuffix === '/' ? '' : angularUrlSuffix;
+    this.window.location.href = `${environment.djangoRoot}${djangoUrlSuffix}`;
```

Comment:
I'm just a bit worried about device agnosticism, but if that is the case I think your solution is good enough :)